### PR TITLE
Add local driver

### PR DESCRIPTION
**Pull request by maxsmythe**
Signed-off-by: Max Smythe <smythe@google.com>

### CODE REVIEW

**Comment by maxsmythe** [comment]
Almost done. Probably need to fix up the `regolib` package and write unit tests for client and integration tests for the remote client.

File: constraint/pkg/client/drivers/local/local.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	for k, v := range modules {
+		if filter != "" && k == filter {
+			continue
+		}
+		m[k] = v
+	}
+	return m
+}
+
+func (d *driver) PutRule(ctx context.Context, name string, src string) error {
```
Comment (tsandall) [inline constraint/pkg/client/drivers/local/local.go]: Nit: unrelated to the main content of this PR, but perhaps this should be called PutModule instead?

Comment (maxsmythe) [reply constraint/pkg/client/drivers/local/local.go]: I like it! Changed.

**Comment by tsandall** [comment]
LGTM

**Pull request merged**